src/views/components/building-address/attributes.njk

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
I wonder if it would be helpful to have a README in this folder just explaining that this component is an extension of the govuk-input https://github.com/alphagov/govuk-frontend/tree/main/packages/govuk-frontend/src/govuk/components/input ?

Reply:
https://github.com/govuk-one-login/ipv-cri-address-front/pull/1008/commits/69b82946b89bd84e29881637ca5e4eeba3545647